doc/manual/parallel-computing.rst

```diff
@@ -858,10 +858,10 @@ Custom cluster managers would typically specify only ``io`` or ``host`` / ``port
 - ``count``, ``exename`` and ``exeflags`` are relevant for launching additional workers from a worker.
   For example, a cluster manager may launch a single worker per node, and use that to launch
   additional workers.
-    - ``count`` with an integer value ``n`` will launch a total of ``n`` workers.
-    - ``count`` with a value of ``:auto`` will launch as many workers as cores on that machine.
-    - ``exename`` is the name of the ``julia`` executable including the full path.
-    - ``exeflags`` should be set to the required command line arguments for new workers.
+  - ``count`` with an integer value ``n`` will launch a total of ``n`` workers.
```

Comment:
Yup, the newlines above and below are needed, rst can be quite picky sometimes. (Sorry about the conflicts.)